src/comm/aca_grpc.cpp

```diff
@@ -76,7 +76,7 @@ void GoalStateProvisionerAsyncServer::RunServer(int thread_pool_size)
 
   thread_pool_.resize(thread_pool_size);
   /* wait for thread pool to initialize*/
-  while (thread_pool_.n_idle() != thread_pool_.size());
+  while (thread_pool_.n_idle() != thread_pool_.size()) {};
```

Comment:
please add a few ms sleep in each loop, otherwise it will be cpu clock time for each loop which is too frequent?

Reply:
Fixed